ui/lib/core/addon/components/form-field.hbs

```diff
@@ -80,8 +83,54 @@
           {{/if}}
         </Hds::Form::Select::Field>
       {{/if}}
+    {{else}}
+      {{#if (or (eq @attr.type "number") (eq @attr.type "string"))}}
+        {{#if (eq @attr.options.editType "password")}}
+          <Hds::Form::TextInput::Field
+            @type="password"
+            {{! about this visibility toggle, see: https://hashicorp.atlassian.net/browse/VAULT-34870 }}
```

Comment:
Not something you need to change here, but for future, if there is a comment we intend to remove in the future we add a TODO. And because there can be a lot of todos, you can use your initials. For me, I use ARG TODO (angel rae gar..)

Reply:
Done